File: proyect-master/app/src/main/java/com/example/myloginapp/MainActivity.java
package com.example.myloginapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import com.google.android.material.button.MaterialButton;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);



        TextView username =(TextView) findViewById(R.id.username);
        TextView password =(TextView) findViewById(R.id.password);

        MaterialButton loginbtn = (MaterialButton) findViewById(R.id.Entrar);

        //usuario y contra admin admin

        loginbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if(username.getText().toString().equals("Jose161610") && password.getText().toString().equals("123424")){
                    //correct
                    Toast.makeText(MainActivity.this,"Sesion iniciada!",Toast.LENGTH_SHORT).show();
                }else
                    //incorrect
                    Toast.makeText(MainActivity.this,"Error en los datos",Toast.LENGTH_SHORT).show();
            }

        });





    } }

File: PokeApp2/app/src/main/java/com/example/pokeapp/MainActivity.kt
package com.example.pokeapp

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val username = findViewById<View>(R.id.username) as TextView
        val password = findViewById<View>(R.id.password) as TextView
        val loginbtn = findViewById<View>(R.id.loginbtn) as MaterialButton

        //usuario y contra
        loginbtn.setOnClickListener {
            if (username.text.toString() == "jose" && password.text.toString() == "123424") {
                //correct

                Toast.makeText(this@MainActivity, "Sesion inicia,  redirigiendo", Toast.LENGTH_SHORT)
                    .show()
                var intent = Intent(this, ListEstados::class.java)
                startActivity(intent)
            } else  //incorrect
                Toast.makeText(
                    this@MainActivity,
                    "Algo anda mal, Favor de comprobar sus datos.!!!",
                    Toast.LENGTH_SHORT
                ).show()
        };


    }
}